editor/plugins/animation_blend_tree_editor_plugin.cpp

```diff
@@ -170,7 +170,13 @@ void AnimationNodeBlendTreeEditor::update_graph() {
 			name->connect("text_changed", callable_mp(this, &AnimationNodeBlendTreeEditor::_node_rename_lineedit_changed), CONNECT_DEFERRED);
 			base = 1;
 			agnode->set_closable(true);
-			node->connect("delete_request", callable_mp(this, &AnimationNodeBlendTreeEditor::_delete_node_request).bind(E), CONNECT_DEFERRED);
+
+			Button *delete_button = memnew(Button);
```

Comment:
This change is good, but I think it would make sense to check for `if (!read_only) {` only see the delete button only appears when the editor is not operating in read-only mode?